### ISSUE TITLE
[RFC] Integration with monolog

### ISSUE THREAD

**Issue by jderusse** [ekino/EkinoNewRelicBundle#143]
Monolog provides an handler to report logged errors to NewRelic which is from my experience more usefull than `log_exceptions`. But is not easily configurable (log level, disable when extension is not loaded, ...)

What's about providing an "shortcut" in this bundle to simplify this configuration?
 - provides a new config parameter log_logs (i'm not sure about the name....)
 - customize log level and log channel
 - check if Monolog is installed
 - inject the well configured handler in monolog
 - check if the extension is loaded (could also wrap the handler in a class who check it at runtime)

```yaml
ekino_new_relic:
  log_logs: (treat like canBeEnabled)
    channel: ~ (default null, when null, meaning injected into `monolog.logger`
    level: !php/const \Monolog\Logger::ERROR
```

If you like the idea, I'm going to implement it.

**Comment by Nyholm** [comment]
So we basically provide some config and configure the handler? 

Im not sure we are going to create a handler, but if the handler already exists somewhere else I think it makes sense to allow easy configuration for it.

**Comment by jderusse** [comment]
Yep exactly, this is just a shortcut to configure monolog's handler (it's not basically the same config has monolog don't allow to figure level and this handler)

**Comment by Nyholm** [comment]
That would be great

**Comment by Nyholm** [comment]
Closed by #145
